pom.xml

```diff
@@ -731,11 +731,6 @@
       </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
-        <artifactId>grpc-google-common-protos</artifactId>
```

Comment:
I don't remember what this is for, but are we sure we can delete this? Datastore's pom.xml declares dependency on it but nowhere else.

Reply:
I checked the contents, and it is *identical* to grpc-google-longrunning-v1. Thus, we aren't losing anything at all. 

Datastore declares a dependency on it because that was the original package name we used. Later we split out proto-google-common-protos, which contained the protos that datastore *actually* needed. Datastore doesn't need the long-running Grpc stub.